requirements.txt

```diff
@@ -1,6 +1,4 @@
-setuptools-markdown
-ConfigArgParse
-mock
-nose
-six
-vcrpy
\ No newline at end of file
+-e .
```

Comment:
Interesting way to install dependencies. I didn't see this before. Did you know any existing project using this approach? Generally speaking, we want to avoid `pip install -e` because it has many unwanted outcomes.